sdk/tests/conformance2/extensions/ext-texture-norm16.html

```diff
@@ -174,6 +189,11 @@
   testNorm16Render(ext.R16_EXT, gl.RED, gl.UNSIGNED_SHORT, 0);
   testNorm16Render(ext.RG16_EXT, gl.RG, gl.UNSIGNED_SHORT, 0);
   testNorm16Render(ext.RGBA16_EXT, gl.RGBA, gl.UNSIGNED_SHORT, 0);
+
+  testSnorm16Unrenderable(ext.R16_SNORM_EXT, gl.RED, gl.SHORT);
+  testSnorm16Unrenderable(ext.RG16_SNORM_EXT, gl.RG, gl.SHORT);
+  testSnorm16Unrenderable(ext.RGB16_SNORM_EXT, gl.RGB, gl.SHORT);
```

Comment:
https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_render_snorm.txt doesn't include RGB16_SNORM
But since it's negtive test, I think it be okay as long as this wouldn't lead to crash.

Reply:
If it crashes, that would make this an essential negative test! :)

Reply:
Thanks for pointing that out. It's probably an oversight in EXT_render_snorm; there is a reference to RGB16_SNORM_EXT in the section "Interactions with EXT_texture_norm16".